runtime/src/comm/ofi/comm-ofi-oob-pmi.c

```diff
@@ -92,9 +92,9 @@ void chpl_comm_ofi_oob_allgather(void* mine, void* all, int size) {
   for (int g_i = 0; g_i < chpl_numNodes; g_i++) {
     char* g_pa = (char*) g_all + g_i * g_size;
     int i;
-    memcpy(&i, g_pa + offsetof(gather_t, nodeID), sizeof(i));
+    chpl_memmove(&i, g_pa + offsetof(gather_t, nodeID), sizeof(i));
```

Comment:
This cannot overlap; `i` is a local on the stack and `g_pa` points to memory just allocated as part of `g_all`.